bundles/org.openhab.binding.miio/src/test/java/org/openhab/binding/miio/internal/ReadmeHelper.java

```diff
@@ -283,14 +312,57 @@ private MiIoBasicDevice findDatabaseEntry(String deviceName) {
         return arrayList;
     }
 
+    private Map<String, String> createI18nEntries() {
+        Map<String, String> i18nEntries = new HashMap<>();
+        String path = "./src/main/resources/database/";
+        File dir = new File(path);
+        File[] filesList = dir.listFiles();
+        for (File file : filesList) {
+            if (file.isFile() && file.getName().toLowerCase().endsWith(".json")) {
+                try {
+                    String key = file.getName().toLowerCase().split("json")[0];
+                    JsonObject deviceMapping = convertFileToJSON(path + file.getName());
+                    Gson gson = new GsonBuilder().serializeNulls().create();
+                    @Nullable
+                    MiIoBasicDevice devdb = gson.fromJson(deviceMapping, MiIoBasicDevice.class);
+                    if (devdb != null) {
+                        for (MiIoBasicChannel channel : devdb.getDevice().getChannels()) {
+                            i18nEntries.put("ch." + key + channel.getChannel(), channel.getFriendlyName());
```

Comment:
The "ch." prefix is also used in **MiIoBasicHandler.addChannel**. Consider adding a constant for it to be reused.

Reply:
yes, fully agree. Done